mh/state.py

```diff
@@ -404,7 +570,11 @@ def get_session(self, pat_ticket):
         """Returns existing PAT session or None."""
         session = self.sessions.get(pat_ticket)
         if session and session.capcom_id:
-            self.use_capcom_id(session, session.capcom_id, session.hunter_name)
+            try:
+                self.use_capcom_id(session, session.capcom_id, session.hunter_name)
+            except Exception as e:
+                print(e)
+                return None
```

Comment:
Why do you shadow the `Exception`? Which exception is raised?

Reply:
Fixed, this has been changed to handle the exception `AssertionError`